<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdePreferencesDialog</class>
 <widget class="QDialog" name="IdePreferencesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidgetPreferences">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxWelcomeDialog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Welcome Dialog</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="checkBoxShowWelcomeForm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Show on start</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxClosingWelcomeFormClosesEditor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Closing welcome form closes editor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearRecentProjects">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Recent Projects</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdateFileAssociations">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update File Associations</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxShellWindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Shell Window</string>
     </property>
     <widget class="QCheckBox" name="checkBoxToggleShellOnRun">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Toggle visibility on run</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxToggleShellOnScene">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Toggle visibility when opening a scene</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabEditors">
    <attribute name="title">
     <string>Editors</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabThemes">
    <attribute name="title">
     <string>Themes</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Compiler</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxCompilerDisassembly">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Print disassembly after compiling</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IdePreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IdePreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
